src/start_server.ts

```diff
@@ -100,11 +100,13 @@ export function getApp(options: ServerOptions) {
 
   app.use('/components/', polyserve);
 
+  const filePathRegex: RegExp = /.*\/.+\..{1,}$/;
```

Comment:
nit: can you move this above ~L86 or below ~L109? Stylistically it feels odd here in between our two route definitions.